pkg/kotsadm/objects/scripts/import-minio-data.sh

```diff
@@ -50,8 +44,14 @@ until $KOTSADM_MINIO_MIGRATION_DIR/bin/mc alias set $KOTSADM_MINIO_NEW_ALIAS $KO
     sleep 1
 done
 
+# wait for minio to be ready
+until $KOTSADM_MINIO_MIGRATION_DIR/bin/mc ready $KOTSADM_MINIO_NEW_ALIAS; do
+    echo "waiting for minio to be ready"
+    sleep 1
+done
+
 # check if the bucket already exists
-if $KOTSADM_MINIO_MIGRATION_DIR/bin/mc ls $KOTSADM_MINIO_NEW_ALIAS | grep -q $KOTSADM_MINIO_BUCKET_NAME; then
+if $KOTSADM_MINIO_MIGRATION_DIR/bin/mc ls $KOTSADM_MINIO_NEW_ALIAS/$KOTSADM_MINIO_BUCKET_NAME; then
     echo "bucket already exists, skipping creation"
```

Comment:
newer minio images do not include the `curl` or `grep` utilities, so these checks were changed to leverage the `mc` binary instead